pyfar/constants/speed_of_sound.py

```diff
@@ -0,0 +1,41 @@
+"""File containing all speed of sound calculation functions."""
+import numpy as np
+
+
+def speed_of_sound_simple(temperature):
+    r"""
+    Calculate the speed of sound in air using a simplified version
+    of the ideal gas law based on the temperature.
+
+    Calculation is in accordance with ISO 9613-1 [#]_, as described in
+    ISO 17497-1 [#]_.
+
+    .. math::
+
+        c = 343.2 \cdot \sqrt{\frac{t + 273.15}{293.15}} m/s
+
+    Parameters
+    ----------
+    temperature : float, array_like
+        Temperature in degree Celsius from -20°C to +50°C.
+
+    Returns
+    -------
+    speed_of_sound : float, array_like
+        Speed of sound in air in (m/s).
+
+    References
+    ----------
+    .. [#] ISO 9613-1:1993, Acoustics -- Attenuation of sound during
+           propagation outdoors -- Part 1: Calculation of the absorption of
+           sound by the atmosphere.
+    .. [#] ISO 17497-1:2004, Acoustics -- Sound-scattering properties of
+           surfaces -- Part 1: Measurement of the random-incidence scattering
+           coefficient in a reverberation room.
```

Comment:
Is this reference needed?
```suggestion
```

Reply:
the fomula is not mentioned in 9613-1, its just mentioned in ISO 17497-1:2004 and refering to 9613-1. threfore I have added both.

Reply:
Thanks for the info! As both the T0=293.15 K and c = 343,2 (T/T0)^1/2 are also mentioned in 9613, I think it makes sense reformulate as suggested above.